libs/aws/tests/integration_tests/agents/test_bedrock_agents.py

```diff
@@ -0,0 +1,411 @@
+import json
```

Comment:
We don't have the permissions to set up the agents and action groups. Can we add the @skip decorator for the tests now.

Reply:
done.